portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,69 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Angeline Chen</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons">
+    <link rel="stylesheet" href="https://unpkg.com/bootstrap-material-design@4.1.1/dist/css/bootstrap-material-design.min.css" integrity="sha384-wXznGJNEXNG1NFsbm0ugrLFMQPWswR3lds2VeinahP8N0zJw9VWSopbjv2x7WCvX" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+  <body onload="getData()">
+    <header>
+        <div class="animation jumbotron">
+            <img src="images/Photo.jpeg" width="500" />
+            <h1 class="display-4">Angeline Chen</h1>
+            <div class="d-flex justify-content-around d-flex align-items-center">
+                <a href="https://www.linkedin.com/in/angeline-chen" target="_blank"><i class="fab fa-linkedin fa-3x"></i></a>
+                <a href="https://github.com/angelinechen9" target="_blank"><i class="fab fa-github fa-3x"></i></a>
+            </div>
+        </div>
+    </header>
+    <main>
+        <div class="card bg-white text-dark border-white">
+            <div class="card-body">
+                <h1 class="card-text">About Me</h1>
```

Comment:
rather than add class="card-text" to every element in here, you could add a CSS rule at a higher level, something like
```
card-body h1,
card-body h2,
card-body h3,
card-body p {
  /* card-text styles */
}
```

would allow you to cut down on the repeated code

Reply:
The class "card-text" is a Bootstrap class.

Reply:
got it. disregard then :)